src/components/MediaCollection/createTrackMediaEvent.js

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import MediaEvents from "./constants/eventTypes";
+
+export default ({ mediaEventManager, mediaSessionCacheManager, config }) => {
+  const sendMediaEvent = options => {
+    const event = mediaEventManager.createMediaEvent({ options });
+    const { playerId, xdm } = options;
+    const eventType = xdm.eventType;
+    const action = eventType.split(".")[1];
```

Comment:
Do we do any validation on the eventType? What happens if it's not set, or if it is set to something without a "."?